frontend/src/components/layout/SignUp.jsx

```diff
@@ -0,0 +1,40 @@
+import styles from "./SignUp.module.css";
```

Comment:
This import does not work when I run `npm run dev` because the file is `Signup` when this imports `SignUp`. Let's name the file `SignUp.module.css` to be consistent with the component file name.